json_logging/json_logging/__init__.py

```diff
@@ -161,11 +198,32 @@ def format(self, record: logging.LogRecord) -> str:
 }
 
 
-def configure_logging() -> None:
+def configure_logging(level: Union[int, str] = "INFO") -> None:
+    """Configure logging module to use JSON formatter for logs."""
     logging.config.dictConfig(LOGGING_STREAM_CONFIG)
+    logging.captureWarnings(True)
+    logging.root.setLevel(level)
+
+
+# Just for developer convenience -- this avoids having too many imports of "logging" packages.
+def getLogger(name: str = None) -> logging.Logger:
+    return logging.getLogger(name)
+
+
+def set_output_format(pretty: bool = False, pretty_if_tty: bool = False) -> None:
+    if pretty or (pretty_if_tty and sys.stdout.isatty()):
```

Comment:
As @willcampbell433 pointed out and from testing in augur, when doing `sls invoke local` with `set_output_format(pretty_if_tty=True)` it does not work. Neither does it work in jupyter notebooks.

Reply:
That is intentional.
```
set_output_format(pretty_if_tty=True)
```
This allows you to deploy this using `sls deploy` and you still don't end up with a "pretty" format _accidentally_ in CloudWatch logs.

Please note that `sls invoke local` is supposed to give you the same context as a run in AWS Lambda. So the fact that output logging doesn't expand into the pretty form is proof that the `pretty_if_tty` flag **is** working as intended.

If you are working locally and are 100% sure that you won't deploy this to AWS Lambda, then you can change the line to:
```
set_output_format(pretty=True)
```
The same is true for Jupyter notebooks. Simply turn the test on "is tty" off.

An option that you can implement in your code is to use something like:
```
pretty = ast.literal_eval(os.getenv("USE_PRETTY_LOGGING", "False"))
set_output_format(pretty=pretty)
```
Then have `export USE_PRETTY_LOGGING=True` in your shell.